src/registrar/templates/application_anything_else.html

```diff
@@ -2,7 +2,9 @@
 {% load field_helpers %}
 
 {% block form_instructions %}
-  <p>Is there anything else you'd like us to know about your domain request? This question is optional.</p>
+    <h2>Is there anything else you'd like us to know about your domain request?</h2> 
```

Comment:
change prime mark to apostrophe in "you'd"

Reply:
```suggestion
    <h2>Is there anything else you’d like us to know about your domain request?</h2> 
```